src/main/java/me/magnet/consultant/ServiceIdentifier.java

```diff
@@ -123,6 +138,7 @@ public String toString() {
 		getDatacenter().ifPresent(dc -> descriptors.add("dc=" + dc));
 		getHostName().ifPresent(host -> descriptors.add("host=" + host));
 		getInstance().ifPresent(instance -> descriptors.add("instance=" + instance));
+		getTags().stream().forEach((tag) -> descriptors.add("tag=" + tag));
```

Comment:
Well ideally when no tags are set it shouldn't be shown at all. The same principle applies to the other variables in the `ServiceIdentifier`.